clang/test/CodeGenSYCL/debug-info-srcpos-kernel.cpp

```diff
@@ -1,5 +1,8 @@
 // RUN: %clang -fno-sycl-force-inline-kernel-lambda -fsycl-device-only %s -S -emit-llvm -O0 -g -o - | FileCheck %s
 //
+// XFAIL: *
```

Comment:
My changes have triggered a test run that apparently fails in general. I cloned a fresh repo from the sycl branch and verified it locally. I opened an issue and adding an XFAIL linking it here.

Reply:
I'm not sure how did that happen?

Reply:
This test seems to be skipped in a lot of PR's. I happened to touch files that have triggered a rerun of it and apparently something has changed in the compiler to cause this to fail.

Reply:
Turned out to be false negative due to my local config. Removed it.